pydy/viz/scene.py

```diff
@@ -24,24 +25,20 @@
 
 try:
     import IPython
-    from IPython.html import widgets
-    from IPython.display import display, Javascript
-
-    ipy_ver = parse_version(IPython.__version__)
-    if ipy_ver < parse_version('2.0'):
-        raise ImportWarning('PyDy supports IPython >= 2.0.0, while you have ' +
-                            'IPython ' + IPython.__version__ + ' installed. ' +
-                            'IPython related functionalities will not be ' +
-                            'available')
-        ipython_less_than_3 = None
-    elif ipy_ver >= parse_version('2.0') and ipy_ver < parse_version('3.0'):
+    if parse_version(IPython.__version__) < parse_version('3.0'):
+        msg = ('PyDy only supports IPython >= 3.0.0. You have '
+               'IPython {} installed. IPython related functionalities will '
+               'not be available')
+        with warnings.catch_warnings():
```

Comment:
The reason I used this way of setting the filter was because if you set the warnings filter globally in a module, you'll turn on all warnings for that python session, including, I believe, all the warnings that are normally hidden in any external module you import. Maybe this was a problem when we were testing against NumPy 1.6 and SciPy 0.9.